platform/app/src/components/ViewportGrid.tsx

```diff
@@ -327,6 +340,9 @@ function ViewerViewportGrid(props) {
               viewportOptions={viewportOptions}
               displaySetOptions={displaySetOptions}
               needsRerendering={displaySetsNeedsRerendering}
+              onReady={() => {
```

Comment:
That is quite nice to have, to get a callback that is timed for actually being ready.